scripts/sv_spark_pipeline_launch.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+if [[ -z ${1+x} ]]; then
+    echo "please provide address for output directory you have write access to"
+    exit 1
+else
+    OUTPUT=$1
+    echo "output directory is set by user to '$1'"
+fi
+
+# STAGE 1 args
+ONE_MEGA_BYTE=1000000
+TEN_MEGA_BYTES=10000000
```

Comment:
Counting repetitive sequences is difficult for me, that's why I put in these names so if we decide to switch the argument, I don't have to count them again...